package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "test": "jest",
     "build": "webpack",
-    "test:coverage": "npm test -- --coverage",
+    "test:coverage": "npm test -- --coverage && cat ./coverage/lcov.info | coveralls",
     "test:badges": "npm run test:coverage  && jest-coverage-badges",
```

Comment:
@christyjacob4 If we are using coveralls for badge, I think we can get rid off jest-coverage-badges.

Reply:
Yes let me remove "test:badges"